VMEncryption/main/CommandExecutor.py

```diff
@@ -55,8 +58,19 @@ def Execute(self, command_to_execute, raise_exception_on_failure=False, communic
                     self.logger.log("Process creation failed: " + str(e))
                 return -1
 
-        stdout, stderr = proc.communicate(input=input)
```

Comment:
Do you think it is possible to add a quick unit test for this?
You will have to make a new test file and set up the mock for Popen to test CommandExecutor.py
but it might just be easy enough to make it worthwhile.

Reply:
Unit tests for both success and failure cases (a timely return and a timeout) would increase confidence here.